2025-05-27 10:22:07,923 INFO    Thread-14138 (_run_job):3230201 [wandb_setup.py:_flush():76] Current SDK version is 0.17.4
2025-05-27 10:22:07,923 INFO    Thread-14138 (_run_job):3230201 [wandb_setup.py:_flush():76] Configure stats pid to 3230201
2025-05-27 10:22:07,923 INFO    Thread-14138 (_run_job):3230201 [wandb_setup.py:_flush():76] Loading settings from /home/anwer/.config/wandb/settings
2025-05-27 10:22:07,923 INFO    Thread-14138 (_run_job):3230201 [wandb_setup.py:_flush():76] Loading settings from /12tb_dsk3/SEIR_model/src/wandb/settings
2025-05-27 10:22:07,923 INFO    Thread-14138 (_run_job):3230201 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'sweeps-SEIR', 'entity': 'danish2562022', 'root_dir': '/12tb_dsk3/SEIR_model/src', 'run_id': '79datom4', 'sweep_param_path': '/12tb_dsk3/SEIR_model/src/wandb/sweep-i7kuc4te/config-79datom4.yaml', 'sweep_id': 'i7kuc4te'}
2025-05-27 10:22:07,923 INFO    Thread-14138 (_run_job):3230201 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2025-05-27 10:22:07,923 INFO    Thread-14138 (_run_job):3230201 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2025-05-27 10:22:07,923 INFO    Thread-14138 (_run_job):3230201 [wandb_setup.py:_flush():76] Applying login settings: {}
2025-05-27 10:22:07,923 INFO    Thread-14138 (_run_job):3230201 [wandb_init.py:_log_setup():529] Logging user logs to /12tb_dsk3/SEIR_model/src/wandb/run-20250527_102207-79datom4/logs/debug.log
2025-05-27 10:22:07,923 INFO    Thread-14138 (_run_job):3230201 [wandb_init.py:_log_setup():530] Logging internal logs to /12tb_dsk3/SEIR_model/src/wandb/run-20250527_102207-79datom4/logs/debug-internal.log
2025-05-27 10:22:07,923 INFO    Thread-14138 (_run_job):3230201 [wandb_init.py:_jupyter_setup():475] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f14c6f5c2f0>
2025-05-27 10:22:07,923 INFO    Thread-14138 (_run_job):3230201 [wandb_init.py:init():569] calling init triggers
2025-05-27 10:22:07,923 INFO    Thread-14138 (_run_job):3230201 [wandb_init.py:init():576] wandb.init called with sweep_config: {'ah0': 2.345919879892421, 'ah1': 2.489175437088782, 'bh0': 8.961915373863526, 'bh1': 2.963632554409086, 'c': 1.4618536246445155}
config: {}
2025-05-27 10:22:07,923 INFO    Thread-14138 (_run_job):3230201 [wandb_init.py:init():619] starting backend
2025-05-27 10:22:07,923 INFO    Thread-14138 (_run_job):3230201 [wandb_init.py:init():623] setting up manager
2025-05-27 10:22:07,925 INFO    Thread-14138 (_run_job):3230201 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-05-27 10:22:07,925 INFO    Thread-14138 (_run_job):3230201 [wandb_init.py:init():631] backend started and connected
2025-05-27 10:22:07,929 INFO    Thread-14138 (_run_job):3230201 [wandb_run.py:_config_callback():1382] config_cb None None {'ah0': 2.345919879892421, 'ah1': 2.489175437088782, 'bh0': 8.961915373863526, 'bh1': 2.963632554409086, 'c': 1.4618536246445155}
2025-05-27 10:22:07,930 INFO    Thread-14138 (_run_job):3230201 [wandb_run.py:_label_probe_notebook():1334] probe notebook
2025-05-27 10:22:07,930 INFO    Thread-14138 (_run_job):3230201 [wandb_run.py:_label_probe_notebook():1344] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2025-05-27 10:22:07,930 INFO    Thread-14138 (_run_job):3230201 [wandb_init.py:init():720] updated telemetry
2025-05-27 10:22:07,932 INFO    Thread-14138 (_run_job):3230201 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2025-05-27 10:22:08,323 INFO    Thread-14138 (_run_job):3230201 [wandb_run.py:_on_init():2402] communicating current version
2025-05-27 10:22:08,397 INFO    Thread-14138 (_run_job):3230201 [wandb_run.py:_on_init():2411] got version response upgrade_message: "wandb version 0.19.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2025-05-27 10:22:08,397 INFO    Thread-14138 (_run_job):3230201 [wandb_init.py:init():804] starting run threads in backend
2025-05-27 10:22:09,119 INFO    Thread-14138 (_run_job):3230201 [wandb_run.py:_console_start():2380] atexit reg
2025-05-27 10:22:09,119 INFO    Thread-14138 (_run_job):3230201 [wandb_run.py:_redirect():2235] redirect: wrap_raw
2025-05-27 10:22:09,119 INFO    Thread-14138 (_run_job):3230201 [wandb_run.py:_redirect():2300] Wrapping output streams.
2025-05-27 10:22:09,119 INFO    Thread-14138 (_run_job):3230201 [wandb_run.py:_redirect():2325] Redirects installed.
2025-05-27 10:22:09,119 INFO    Thread-14138 (_run_job):3230201 [wandb_init.py:init():847] run started, returning control to user process
2025-05-27 10:22:09,223 INFO    Thread-14138 (_run_job):3230201 [wandb_run.py:_finish():2109] finishing run danish2562022/sweeps-SEIR/79datom4
2025-05-27 10:22:09,224 INFO    Thread-14138 (_run_job):3230201 [jupyter.py:save_history():444] not saving jupyter history
2025-05-27 10:22:09,224 INFO    Thread-14138 (_run_job):3230201 [jupyter.py:save_ipynb():372] not saving jupyter notebook
2025-05-27 10:22:09,224 INFO    Thread-14138 (_run_job):3230201 [wandb_init.py:_jupyter_teardown():457] cleaning up jupyter logic
2025-05-27 10:22:09,224 INFO    Thread-14138 (_run_job):3230201 [wandb_run.py:_atexit_cleanup():2349] got exitcode: 0
2025-05-27 10:22:09,224 INFO    Thread-14138 (_run_job):3230201 [wandb_run.py:_restore():2332] restore
2025-05-27 10:22:09,224 INFO    Thread-14138 (_run_job):3230201 [wandb_run.py:_restore():2338] restore done
2025-05-27 10:22:12,367 INFO    Thread-14138 (_run_job):3230201 [wandb_run.py:_footer_history_summary_info():3991] rendering history
2025-05-27 10:22:12,367 INFO    Thread-14138 (_run_job):3230201 [wandb_run.py:_footer_history_summary_info():4023] rendering summary
2025-05-27 10:22:12,369 INFO    Thread-14138 (_run_job):3230201 [wandb_run.py:_footer_sync_info():3950] logging synced files
